README.md

```diff
@@ -16,10 +16,16 @@
 
 A cron job wrapper to add some missing features (locks, timeouts, random sleeps, env loading...).
 
+
+## Setup
```

Comment:
README.md file in the repository root is generated from a template

custom changes (for this particular repo) must be made in `.metwork-framework/README.md` file 
(it will be copied here by an external process)

=> can you change your PR on this particular point?

Reply:
Thanks. Sure, I will carry over that change to the other file.